src/core/stdc/stdio.d

```diff
@@ -832,11 +910,12 @@ extern (D) @trusted
     ///
     int getchar()                 { return getc(stdin);     }
     ///
-    int putchar(int c)            { return putc(c,stdout);  }
+    int putchar(int c)            { return putc(c, stdout);  }
+
     ///
     int getc(FILE* stream)        { return fgetc(stream);   }
     ///
-    int putc(int c, FILE* stream) { return fputc(c,stream); }
+    int putc(int c, FILE* stream) { return fputc(c, stream); }
```

Comment:
I don't see the point of these two spacing tweaks.

Reply:
Reverted